docs/ide/copilot-agent.md

```diff
@@ -0,0 +1,158 @@
+---
+title: 'GitHub Copilot Agent (Preview)'
+description: GitHub Copilot Agent iterates on code in Visual Studio by making code edits, running commands, and reading error/build context.
+ms.date: 5/8/2025
+ms.topic: overview
+author: aayim
+ms.author: meghaanand
+ms.manager: mijacobs
+ms.subservice: ai-tools
+ms.collection: ce-skilling-ai-copilot
+monikerRange: '>= vs-2022'
+---
+# What is GitHub Copilot Agent?
+
+With GitHub Copilot's agent mode in Visual Studio, you can use natural language to specify a high-level task, and let AI autonomously reason about the request, plan the work needed, and apply the changes to your codebase. Agent mode uses a combination of code editing and tool invocation to accomplish the task you specified. As it processes your request, it monitors the outcome of edits and tools, and iterates to resolve any issues that arise.
+
+The critical difference compared to Copilot Chat is that the Agent mode can:
+
+1) Run commands and builds to interpret the environment or execute a task (i.e. database migration, dotnet restore, etc.)
+2) Iterate on errors, failed builds, unit test results until the agent requires more input or deems the task accomplished.
+
+## Prerequisites
++ [Visual Studio 2022 version 17.14](/visualstudio/releases/2022/release-history) and later have the Copilot Agent.
+    + In the initial 17.14 release, you will need to set `Enable agent mode in the chat pane` in Tools > Options > GitHub > Copilot.
++ [Sign in to Visual Studio using a GitHub account](work-with-github-accounts.md) with [Copilot access](https://docs.github.com/en/copilot/about-github-copilot/what-is-github-copilot#getting-access-to-copilot). <br/>
+    + You can use [GitHub Copilot for Free](copilot-free-plan.md). Sign up and leverage AI to code faster and more efficiently.
+
+## How to use Copilot Agent
+
+In agent mode, Copilot operates autonomously and determines the relevant context for your prompt.
+
+Follow these steps to get started:
+
+1. Make sure that agent mode is enabled by clicking Tools > Options > GitHub > Copilot and  the "Enable Agent mode in the chat pane".
+
+![Copilot Agent Settings](./media/vs-2022/copilot-agent/copilotagent-settingupdated.png)
+
+2. Enable agent mode by opening the GitHub Copilot window, clicking the mode dropdown that says "Ask", then select "Agent".
```

Comment:
Super nit - I'm not sure what's considered "correct", but we have a mix of "agent mode" and "Agent mode" throughout the doc. Just thought I'd mention it since I noticed the inconsistency